### PR TITLE
Added support for larger root volumes

### PR DESCRIPTION
**Pull request by nielsdraaisma**

### CODE REVIEW

File: roles/aws.ec2-autoscaling-group/defaults/main.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ asg_max_size: 1
 asg_desired_capacity: 1
 asg_subnets: []
 asg_launch_config_instance_size: "t2.small"
+asg_launch_config_root_volume_size: 16 # Volume of the root volume on the ecs instance ( in GB )
```
Comment (simplesteph) [inline roles/aws.ec2-autoscaling-group/defaults/main.yml]: should we keep the default at 8, just like it it today for AWS?

Comment (nielsdraaisma) [inline roles/aws.ec2-autoscaling-group/defaults/main.yml]: could do, but 8 is pretty small and we keep hitting this limit. 16gb just gives a bit more space by default.

File: roles/aws.ec2-autoscaling-group/tasks/main.yml [hunk 2]
```diff
@@ -65,6 +72,7 @@
       user_data: "{{ _launch_config_user_data }}"
       instance_profile_name: "{{ asg_launch_config_instance_profile_name }}"
       assign_public_ip: "{{ asg_launch_config_assign_public_ip }}"
+      volumes: "{{ _launch_config_volumes }}"
```
Comment (simplesteph) [inline roles/aws.ec2-autoscaling-group/tasks/main.yml]: I know it's out of scope for this PR, but can you please include "{{ _launch_config_volumes + asg_additional_volumes }}, with default being [] ?

File: roles/aws.ec2-autoscaling-group/defaults/main.yml (hunk 1 above)
Comment (simplesteph) [inline roles/aws.ec2-autoscaling-group/defaults/main.yml]: sounds good - let's just document this somewhere in the readme please

**Pull request merged**